dist/immutable.js

```diff
@@ -15,8 +15,16 @@
   function createClass(ctor, superClass) {
     if (superClass) {
       ctor.prototype = Object.create(superClass.prototype);
+      if (superClass.__proto__) {
+        ctor.__proto__ = superClass
```

Comment:
@tgriesser @dandelany I don't know JavaScript's prototype system well, but is it all right here? Why we check `superClass.__proto__` here but don't use it?